packages/flutter_tools/test/integration.shard/test_data/gen_l10n_project.dart

```diff
@@ -80,45 +103,50 @@ class Home extends StatelessWidget {
             results.add(AppLocalizations.of(context).hello("GB fallback World"));
           },
         ),
+        Builder(
+          builder: (BuildContext context) {
+            results.add('--- General formatting tests ---');
+          },
+        ),
         LocaleBuilder(
           locale: Locale('en'),
           callback: (BuildContext context) {
+            final AppLocalizations localizations = AppLocalizations.of(context);
+            results.addAll(<String>[
+              '${localizations.helloWorld}',
+              '${localizations.hello("World")}',
+              '${localizations.greeting("Hello", "World")}',
+              '${localizations.helloWorldOn(DateTime(1960))}',
+              '${localizations.helloOn("world argument", DateTime(1960), DateTime(1960))}',
+              '${localizations.helloWorldDuring(DateTime(1960), DateTime(2020))}',
+              '${localizations.helloFor(123)}',
+              '${localizations.helloCost("price", 123)}',
+              '${localizations.helloWorlds(0)}',
+              '${localizations.helloWorlds(1)}',
+              '${localizations.helloWorlds(2)}',
+              '${localizations.helloAdjectiveWorlds(0, "new")}',
+              '${localizations.helloAdjectiveWorlds(1, "new")}',
+              '${localizations.helloAdjectiveWorlds(2, "new")}',
+              '${localizations.helloWorldsOn(0, DateTime(1960))}',
+              '${localizations.helloWorldsOn(1, DateTime(1960))}',
+              '${localizations.helloWorldsOn(2, DateTime(1960))}',
+              '${localizations.helloWorldPopulation(0, 100)}',
+              '${localizations.helloWorldPopulation(1, 101)}',
+              '${localizations.helloWorldPopulation(2, 102)}',
+              '${localizations.helloWorldsInterpolation(123, "Hello", "World")}',
+              '${localizations.singleQuote}',
+              '${localizations.doubleQuote}',
+            ]);
+          },
+        ),
+        Builder(
+          builder: (BuildContext context) {
             int n = 0;
-            try {
```

Comment:
this was set up with a try/finally so that if an exception was thrown, the test would still finish normally and then fail. Otherwise it would hang waiting for the l10n END string.

Reply:
This would mean that every test case should have a try/finally block right?

Reply:
Just ensure that no matter what happens, the test prints the l10n END string before exiting